tests/src/FunctionalJavascript/MediaAvPortalCreateContentTest.php

```diff
@@ -164,20 +163,28 @@ public function testAvPortalPhotoMediaEntity(): void {
     $this->assertContains('ec.europa.eu/avservices/avs/files/video6/repository/prod/photo/store/', $image_url);
     $this->assertContains('P038924-352937.jpg', $image_url);
 
-    // Edit the newly created media.
-    $this->drupalGet('media/1/edit');
+    $photo_urls = [
+      'https://audiovisual.ec.europa.eu/en/album/M-090909/P-039162~2F00-12',
+      'https://audiovisual.ec.europa.eu/en/photo/P-039162~2F00-12',
+    ];
 
-    // Update the field.
-    $page->fillField('Media AV Portal Photo', 'https://audiovisual.ec.europa.eu/en/photo/P-039162~2F00-12');
-    $page->pressButton('Save');
+    // We could update photo with different URL formats supporting.
```

Comment:
I think this comment should go before the $photo_urls variable is set and a better phrasing would be: "We need to support both individual photos and photos inside albums"